docs:  consul-k8s uninstall with namespace (#11478)
hashicorp/consul: website/content/docs/k8s/operations/uninstall.mdx

Run the `helm uninstall` **and** manually remove resources that Helm does not delete.

> 1. *Optional* : Set your kubeConfig context to the `consul` namespace to avoid having to append `-n consul` to all of the subsequent commands if you've installed Consul in a dedciated namespace.
> 
>     ```
>     kubectl config set-context --current --namespace=consul
>     ```
> 
> 1. Next run `helm uninstall <release-name>` to uninstall Consul using the release name you've installed Consul with:

Reviewer: ```suggestion
1. (Optional) If Consul is installed in a dedicated namespace, set the kubeConfig context to the `consul` namespace. Otherwise, subsequent commands will need to include `-n consul`. 

    ```
    kubectl config set-context --current --namespace=consul
    ```

1. Run the `helm uninstall <release-name>` command and specify the release name you've installed Consul with, e.g.,:
```
Just trying to use a consistent voice and style.
